commands.pm

```diff
@@ -285,15 +285,8 @@ sub run_daemon {
     }
     my $daemon = Mojo::Server::Daemon->new(app => app, listen => ["http://$address:$port"]);
     $daemon->silent;
-    # We need to override the default logging format
-    app->log->format(
-        sub {
-            my ($time, $level, @lines) = @_;
-            # Unfortunately $time doesn't have the precision we want. So we need to use Time::HiRes
-            $time = gettimeofday;
-            return sprintf(strftime("[%FT%T.%%03d %Z] [$level] ", localtime($time)), 1000 * ($time - int($time))) . join("\n", @lines, '');
-        });
-
+    # Use same log format as isotovideo
+    app->log->format(\&bmwqemu::log_format_callback);
```

Comment:
Seems I overlooked that when I introduced Time::Moment. This is great! You can consider to pull out https://github.com/os-autoinst/os-autoinst/pull/1862/commits/40d2a77d18ecc37eacf37c2042e8c0cf86316da5 into its own PR